app/controllers/profiles/accounts_controller.rb

```diff
@@ -4,6 +4,8 @@
 module Profiles
   # Controller for the user account page
   class AccountsController < Profiles::ApplicationController
+    before_action :current_page
```

Comment:
This `before_action :current_page` can probably be moved into the `profiles/application_controller.rb` and can be removed from this controller, `passwords_controller`, `personal_access_tokens_controller`, `preferences_controller`, and `profiles_controller`. The methods to actually set the current page can remain in these controllers. Since we are inheriting from Profiles::ApplicationController we only need the `before_action :current_page` within it.

Reply:
Please see a3b41c47ed02575f26cc4d4f67c869ad194272ff.